samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/App.xaml.cs

```diff
@@ -98,6 +99,8 @@ protected override async void OnLaunched(XamlLaunchActivatedEventArgs e)
 #else
 			_window = XamlWindow.Current;
 #endif
+			var helper = ResponsiveHelper.GetForCurrentView();
+			helper.HookupEvent(_window);

```

Comment:
Is this needed just for the runtime tests? This isn't needed for people to use the control in general, right?

Reply:
HookupEvent is needed to receive SizeChanged event

Reply:
I guess this should be on the docs as well, right @Xiaoy312 ?